File: Pyhton/Bioskop.py
class Bioskop:

    # sebuah konstruktor yang menerima masukan berupa id
    # nama, alamat, jumlah studio dan kota
    def __init__(self, id:int, nama:str, alamat:str, jumlah_studio:int, kota:str):
        self.__id = int(id)
        self.__nama = str(nama)
        self.__alamat = str(alamat)
        self.__jumlah_studio = int(jumlah_studio)
        self.__kota = str(kota)
    #Berbagai method get set
    #Get adalah method untuk mengabil data dari sebuah class
    #set adalah method untuk mengedit atau menambahkan data dari sebuah class
    def getid(self) -> int:
        return self.__id

    def setid(self, id:int) -> None:
        self.__id = int(id)

    def getnama(self) -> str:
        return self.__nama

    def setnama(self, nama:str) -> None:
        self.__nama = str(nama)

    def getalamat(self) -> str:
        return self.__alamat

    def setalamat(self, alamat:str) -> None:
        self.__alamat = str(alamat)

    def getjumlah_studio(self) -> int:
        return self.__jumlah_studio

    def setjumlah_studio(self, jumlah_studio:int) -> None:
        self.__jumlah_studio = int(jumlah_studio)

    def getkota(self) -> str:
        return self.__kota

    def setkota(self, kota:str) -> None:
        self.__kota = str(kota)

    

File: C++/Bioskop.cpp
using namespace std;

//deklarasi kelas baru dengan nama Bioskop
class Bioskop {
private:
    int id;
    string nama;
    string alamat;
    int jumlah_studio;
    string kota;

public:
    Bioskop(){//konstruktor untuk class Bioskop

    }
    /*
    konstruktor juga, konstruktor ini digunakan jika user mau langsung
    memasukan data ke dalam class nya, ibaratnya konstruktor tapi dengan isi nya
    */
    Bioskop(int id, string nama, string alamat, int jumlah_studio, string kota){
        this->id = id;
        this->nama = nama;
        this->alamat = alamat;
        this->jumlah_studio = jumlah_studio;
        this->kota = kota;
    }
    //metode untuk meminta data dan men set atau memasukan data 
    void setid(int id){
        this->id = id;
    }
    int getid(){
        return this->id;
    }
    void setnama(string nama){
        this->nama = nama;
    }
    string getnama(){
        return this->nama;
    }
    void setalamat(string alamat){
        this->alamat = alamat;
    }
    string getalamat(){
        return this->alamat;
    }
    void setjumlah_studio(int jumlah_studio){
        this->jumlah_studio = jumlah_studio;
    }
    int getjumlah_studio(){
        return this->jumlah_studio;
    }
    void setkota(string kota){
        this->kota = kota;
    }
    string getkota(){
        return this->kota;
    }
};